### ISSUE TITLE
Implement building form

### ISSUE THREAD

**Issue by rmn36** [themerrymarlin/CWRUtan#8]

**Comment by zaheelee** [comment]
Just to clarify, this means that we need to create and implement the form used to select what structure to build?

**Comment by rmn36** [comment]
Yes. Talk to David about how the board will work but for now implement the
UI for selection itself.
